donate/core/management/commands/test_compilemessages_ci.py

```diff
@@ -0,0 +1,51 @@
+"""Management command running on CI: Send a message on Slack if an error is found while running compilemessages."""
+
+from django.core.management import call_command
+from django.core.management.base import BaseCommand, CommandError
+from django.conf import settings
+import requests
+
+
+class Command(BaseCommand):
+    help = 'Delete all Page and Image objects from the database along with associated media files.'
```

Comment:
This description doesn't seem accurate

Reply:
lol, I always copy-past a previous command so that's why XD